kubernetes/README.md

```diff
@@ -96,70 +96,59 @@ kubectl set env deploy/op-scim-bridge OP_TLS_DOMAIN=scim.example.com
 
 Replace `scim.example.com` with the fully qualified domain name of the DNS record before running this command. Your SCIM bridge will restart, request a TLS certificate from Let's Encrypt, and automatically renew the certificate on your behalf.
 
-## Step 5: Test your SCIM bridge
+## Step 5: Test your SCIM Bridge
```

Comment:
does not match style guide: https://support.1password.com/style-guide/#s

Reply:
I prefer `SCIM Bridge` too, but our style guide currently says these are OK as they are: https://support.1password.com/style-guide/#1password-scim-bridge:~:text=OK%20to%20use%20the%20SCIM%20bridge%20after%20first%20occurrence.%20For%20possessives%2C%20use%20your%20SCIM%20bridge 🤔

Reply:
It should be lowercase :)

```suggestion
## Step 5: Test your SCIM bridge
```